dropwizard-core/src/main/java/io/dropwizard/setup/Demo.java

```diff
@@ -0,0 +1,34 @@
+package io.dropwizard.auth;
+
+public class Demo {
+    public static void main(String[] args) {
+        String rpository = "testRepo";
```

Comment:
Possible typo in `rpository`, fixes:
Candidate #1 with a confidence 100%
```suggestion
        String repository = "testRepo";
```

<!-- lookout footnote separator -->
_Comment made by the analyzer style-check._